queryparams/params.go

```diff
@@ -249,38 +247,118 @@ func CalculatePageNumber(offset, pageSize int) int {
 	return ((offset + 1) / pageSize) + 1
 }
 
-type Sort int
+type SortKind int
 
 const (
-	Invalid Sort = iota
-	RelDateAsc
-	RelDateDesc
-	TitleAZ
-	TitleZA
+	InvalidSort SortKind = iota
+	PublishedNewest
+	PublishedOldest
+	UpcomingNewest
+	UpcomingOldest
+	CancelledNewest
+	CancelledOldest
+	Alpha
+	ReverseAlpha
 	Relevance
+
+	// Frontend labels
+	Newest           = "date-newest"
+	Oldest           = "date-oldest"
+	AlphaUser        = "alphabetical-az"
+	ReverseAlphaUser = "alphabetical-za"
```

Comment:
Should we try and be consistent? Newest and Oldest are "clean" whereas Alpha and ReverseAlpha are suffixed with `User`. Similar in the backed labels